Source/WebCore/loader/HTTPHeaderField.cpp

```diff
@@ -32,8 +32,8 @@ namespace WebCore {
 
 std::optional<HTTPHeaderField> HTTPHeaderField::create(String&& unparsedName, String&& unparsedValue)
 {
-    StringView strippedName = StringView(unparsedName).stripLeadingAndTrailingMatchedCharacters(RFC7230::isWhitespace<UChar>);
-    StringView strippedValue = StringView(unparsedValue).stripLeadingAndTrailingMatchedCharacters(RFC7230::isWhitespace<UChar>);
+    StringView strippedName = StringView(unparsedName).stripLeadingAndTrailingMatchedCharacters(isASCIITabOrSpace<UChar>);
```

Comment:
It's unfortunate it can't find UChar on its own, but it's beyond my expertise to address that.